tool/settings.re

```diff
@@ -125,6 +141,7 @@ struct Settings {
     reticle_h: float,
     reticle_scale: float,
     reticle_scale_position: bool,
+    log_message_duration: float,
```

Comment:
Do you think that people want to configure how long a log message stays on the screen down to microseconds?